pkgs/development/libraries/qt-5/5.15/patches.nix

```diff
@@ -0,0 +1,43 @@
+{
+qt3d = {name="qt3d-5.15.patch";url="https://invent.kde.org/qt/qt/qt3d.git";fromRev="34171b1d99f55fde1627df3c57eed50480ab2ae7";toRev="7edec6e014de27b9dd03f63875c471aac606a918";sha256="12hrq0arwhfk8blqjrb24rcka9cy2asg0mpfck63khcmjhqyaic9";};
```

Comment:
It would be nice if we could preserve the format or at least add a little white space to make further changes/reviews easier.